CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 
+- Improve JSON config file parser error messages
```

Comment:
```suggestion
Improve JSON config file error messages by making them more explicit.
```

And I think this is better suited in the `Changed` section

Reply:
Personally I wouldn't add a changelog entry for this, but I'm curious what others think here.

Reply:
I'm fine either way

Reply:
I also do not think the change it is worth a changelog entry

Reply:
I also do not think the change it is worth a changelog entry